test/services/editions/db/EditionsDBTest.scala

```diff
@@ -160,6 +162,46 @@ class EditionsDBTest extends FreeSpec with Matchers with EditionsDBService with
         commentFront.collections.length shouldBe 3
       }
     }
+
+    "should allow lookup of issue by collection id" taggedAs UsesDatabase in {
+      withEvolutions {
+        withEvolutions {
+          val skeleton = EditionsIssueSkeleton(
+            ZonedDateTime.of(2019, 9, 30, 0, 0, 0, 0, ZoneId.of("Europe/London")),
+            ZoneId.of("Europe/London"),
+            List(
+              EditionsFrontSkeleton(
+                name = "news/uk",
+                presentation = FrontPresentation(),
+                collections = List(
+                  EditionsCollectionSkeleton(
+                    name = "politics",
+                    prefill = Some(CapiPrefillQuery("magic-politics-query")),
+                    presentation = CollectionPresentation(),
+                    items = List("12345", "23456")
+                  ),
+                )
+              )
+            )
+          )
+
+          val now = OffsetDateTime.of(2019, 7, 16, 17, 51, 44, 0, ZoneOffset.ofHours(1))
+
+          val id = editionsDB.insertIssue("daily-edition",
+            skeleton,
+            User("Billy", "Bragg", "billy.bragg@justice.example.com", None),
+            now
+          )
+
+          val retrievedIssue = editionsDB.getIssue(id).value
+          val collectionsId = retrievedIssue.fronts.head.collections.head.id
```

Comment:
do we want to test `retrievedIssue.fronts.length == 1`?

Reply:
That case is covered by other tests so I don't think it is necessary. What I have done though is inserted a couple of extra issues that shouldn't be selected so we know that this isn't working by virtue of returning the single issue that exists in the DB.